spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  it "is valid with valid attributes" do
+    user = FactoryBot.create(:user)
+    expect(user).to be_valid
+  end
+
+  it "is not valid without a status" do 
+    expect{FactoryBot.create(:user, status: nil)}.to raise_error(ActiveRecord::NotNullViolation)
```

Comment:
why this is one-liner?

Reply:
Model missing validations.
It's not clear what causes this error. we also would check error messages